services/web-app/data/remote-pages.js

```diff
@@ -121,7 +121,14 @@ export const remotePages = {
     }
   },
   'elements/typography/styling-strategies': {
-    filePath: '/src/pages/guidelines/typography/styling-strategies.mdx'
+    filePath: '/src/pages/guidelines/typography/styling-strategies.mdx',
+    params: {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'carbon-platform'
+    }
```

Comment:
carbon-website renamed this page to "Style strategies", should we rename instead, or does it make more sense to pull from carbon-platform? 🤔 
https://github.com/carbon-design-system/carbon-website/commit/94d7e1eedcaf9f7017aaf03ed0a25b8d455f0504
whatever we choose I think we need more changes here because there's a tabs mismatch
<img width="533" alt="image" src="https://user-images.githubusercontent.com/40550942/198103530-fad0925f-c721-4859-8133-eb6fd39b16ee.png">
<img width="591" alt="image" src="https://user-images.githubusercontent.com/40550942/198103571-8457335e-a918-4572-8d00-7f09d0cc8d2f.png">

Reply:
Good catch... no idea MDX file names were changed upstream... let me update the `carbon-platform` branch of website repo in a non-breaking way to see if we can pull this off.